cardano-node/src/Cardano/CLI/Benchmarking/Tx/NodeToNode.hs

```diff
@@ -46,8 +57,47 @@ type SendRecvConnect = WithMuxBearer
                                            NtN.NodeToNodeVersion
                                            CBOR.Term))
 
+instance ToObject SendRecvConnect where
+  toObject MinimalVerbosity _ = emptyObject -- do not log
+  toObject NormalVerbosity (WithMuxBearer _ _) =
+    mkObject ["kind" .= String "SendRecvConnect"]
+  toObject MaximalVerbosity (WithMuxBearer connId _) =
+    mkObject [ "kind"   .= String "SendRecvConnect"
+             , "connId" .= String (T.pack . show $ connId)
+             ]
+
+instance DefineSeverity SendRecvConnect
+
+instance DefinePrivacyAnnotation SendRecvConnect
+
+instance forall m . (m ~ IO) => Transformable Text m SendRecvConnect where
+  -- transform to JSON Object
+  trTransformer StructuredLogging verb tr = trStructured verb tr
+  trTransformer _ _verb _tr = nullTracer
+
+--------------------------------------------------------------------------------------
+
 type SendRecvTxSubmission blk = TraceSendRecv (TxSubmission (GenTxId blk) (GenTx blk))
 
+instance forall blk . (RunNode blk) => ToObject (SendRecvTxSubmission blk) where
+  toObject MinimalVerbosity _ = emptyObject -- do not log
+  toObject NormalVerbosity _ =
+    mkObject ["kind" .= String "SendRecvTxSubmission"]
+  toObject MaximalVerbosity _ =
+    mkObject [ "kind"   .= String "SendRecvTxSubmission"
```

Comment:
we need more info:
* timestamp
* tx id

Reply:
Some examples:

```
{"at":"2020-02-19T09:11:18.59Z","env":"1.5.0:baf2a","ns":["cardano","cli","generate-txs","submit"],"data":{"ackNum":0,"time":"2020-02-19 09:11:18.598503221 UTC","reqNum":3,"kind":"TxSubmissionRecvRequestTxIds"},"app":[],"msg":"","pid":"1146","loc":null,"host":"","sev":"Debug","thread":"96"}
```
```
{"at":"2020-02-19T09:11:18.63Z","env":"1.5.0:baf2a","ns":["cardano","cli","generate-txs","submit"],"data":{"time":"2020-02-19 09:11:18.630378862 UTC","kind":"TxSubmissionSendNBReplyTxIds","txIds":[["txid: txid:7252f9bb",307],["txid: txid:f680bb77",307],["txid: txid:ff2a85b8",307]]},"app":[],"msg":"","pid":"1146","loc":null,"host":"","sev":"Debug","thread":"96"}
```
```
{"at":"2020-02-19T09:11:18.63Z","env":"1.5.0:baf2a","ns":["cardano","cli","generate-txs","submit"],"data":{"time":"2020-02-19 09:11:18.630450099 UTC","kind":"TxSubmissionRecvRequestTxs","txIds":["txid: txid:d0d4058e"]},"app":[],"msg":"","pid":"1146","loc":null,"host":"","sev":"Debug","thread":"96"}
```